lib/deps/upsert.js

```diff
@@ -1,7 +1,67 @@
 'use strict';
 
-var upsert = require('pouchdb-upsert').upsert;
+var PouchPromise;
```

Comment:
This can probably be replaced by a ``require()`` that directly gives you the promise object ('./promise').